app/src/main/java/org/oppia/app/options/ReadingTextSizeFragmentPresenter.kt

```diff
@@ -114,15 +120,17 @@ class ReadingTextSizeFragmentPresenter @Inject constructor(private val fragment:
   }
 
   fun getReadingTextSizeInFloat(readingTextSize: ReadingTextSize): Float {
+    val defaultReadingTextSizeInFloat = fragment.context?.resources!!.getDimension(
```

Comment:
Prefer not to use `!!`.
```
    val defaultReadingTextSizeInFloat = fragment.requireContext().resources.getDimension(
      R.dimen.default_reading_text_size
    )
```

Reply:
Yes, actually I did that because if I don't use it, defaultReadungTextSize becomes nullable.